utils.py

```diff
@@ -150,3 +150,6 @@ def Make1DHists(truth, predict, PUPPI, xmin=0, xmax=400, nbins=100, density=Fals
     plt.ylabel(yname)
     plt.savefig(outputname)
     plt.close()
+    
+def to_np_array(ak_array, maxN=100, pad=0):
```

Comment:
reuse this function in `convert.py`

Reply:
You're saying I should just define in `convert.py`, right?

Reply:
No, you can keep the definition here but just import it and use it in `convert.py`